docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- Set SERIALx_PROTOCOL to "RCIN" (23).
+- Set RC_PROTOCOLS to 536.
+- Set RSSI_TYPE to 3.
+
+## 4. Install firmware if required
+
+If both your Rx and Tx modules came pre-installed with mLRS firmware as with the MatekSys mLRS modules, you can use the existing firmware unless you need a bug fix or feature which was added in a newer firmware version.
+
+Carefully install your Tx module into the bay of your EdgeTX or OpenTX radio.  Some Tx modules can be fussy about pin alignment;  don't use excessive force or you may break something.
+
+For the R9M Tx module, it is recommended to [Flash via EdgeTx/OpenTx](FRSKY_R9.md#flash-the-r9m-module-with-elrs-bootloader) which is a 2 step process:
+
+- First, [Flash the ELRS Bootloader](FRSKY_R9.md#flash-the-elrs-bootloader).
+- Next, use your EdgeTX/OpenTX radio to [Flash the mLRS Firmware](FRSKY_R9.md#flashupdate-the-mlrs-firmware).
+
+To flash ELRS receivers, you can use esptool or a Chrome or Chromium based browser as instructed [here](ELRS_RECEIVERS.md#flashing)
+
+[Flashing firmware](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-3) to the MatekSys mLRS Tx modules is especially easy.
+
+Once the firmware is installed, you can confirm operation by observing the LED.
+- When either a receiver or Tx module is powered on by itself, a LED (usually red) should flash at 2 Hz (twice per second, on for 1/4 second then off for 1/4 second repeatedly).
+- When the Tx module and receiver are both powered on and bound, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
+
+## 5. Install the mLRS Lua script and configure your mLRS system
+
+Follow the [instructions](LUA.md) and copy the correct mLRS Lua script to the SCRIPTS\TOOLS folder on your SD card and setup your RC radio external module.
+
+Start mLRS Configurator Lua script from the SYS->TOOLS menu and confirm you can talk to the Tx module.  You should see the TX module firmware version reported at the top of the screen.
+
+Use the mLRS Lua script to Confirm the receiver is connected and the firmware versions are recent enough to support the features you need.
+- If the radio link does not connect, follow the [Binding](BINDING.md) procedure.
+- It is HIGHLY recommended to use the same firmware version on both the receiver and Tx module. The latest released firmware is also recommended, especially for new installs.  If your Rx and Tx module firmware versions don't match or are very much out of date, return to the previous step and install the latest mLRS firmware release.
+
+Use the mLRS Lua script to configure your mLRS system
+- Leave your Rx powered on and connected while you complete the following steps.
+- Set the correct RF Band for your region (If using 900 MHz hardware).
+- You can also change the Bind Phrase if desired.
+- Set your desired Mode
```

Comment:
it seems you want to refere to the parameters
I guess one could also put them all systematically into quotation marks. We also could da as for teh trouble shoot and actually link them to the md, would have the nice side effect they are displayed in nice blue

Reply:
I like the link idea, thanks.